TraceScripts/README.md

```diff
@@ -31,11 +31,11 @@ xbtrace /x:<xboxip> start advancedmp
 xbtrace /x:<xboxip> stop
 ```
 
-## iOS - macOS
+## iOS, macOS, and Android
 
-The PlayFab Multiplayer C++ library for iOS and macOS includes logging capabilities with a configurable verbosity level. Logging configuration is defined in `PlayFabMultiplayerLogger.json` file which can be deployed as an asset along with the application.
+The PlayFab Multiplayer C++ library for iOS, macOS, and Android includes logging capabilities with a configurable verbosity level. Logging configuration is defined in `PlayFabMultiplayerLogger.json` file which can be deployed alongside your application.
 
-When this file is detected by the application at runtime it will use it to enable logging as configured. The following verbosity levels are currently supported:
+When this file is detected by the application (and the PFMultiplayer library) at runtime, it will use it to enable logging as configured. The following verbosity levels are currently supported:
```

Comment:
perhaps sligthly simpler/easier to read?
```suggestion
When this file is detected by the PFMultiplayer library at runtime within your application, it will use it to enable logging as configured. The following verbosity levels are currently supported:
```